index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="website" content="NF website Hugh" />
+    <title>My First NF Website</title>
+
+    <!--   <link href="./style.css" rel="stylesheet" /> -->
+  </head>
+
+  <body>
+    <header>
+      <h1>Hugh Burgess</h1>
+      <nav>
+        <ul>
+          <a href="#Home"><li>Home</li></a>
+          <a href="#Contact"><li>Contact</li></a>
+          <a href="#Career"><li>Career</li></a>
+        </ul>
+      </nav>
+
+      <br />
+
+      <img src="/images/profile-picture.png" alt="Picture of Hugh" />
+    </header>
+    <section>
+      <h2 id="Career">About Me</h2>
+      <article>
+        <p>
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quo, saepe
+          aperiam alias vitae quod autem iusto untur itaque in vel quidem dicta
+          temporibus cupiditate voluptatum numquam velit perferendis maiores
+          animi!
+        </p>
+      </article>
+    </section>
+
+    <section>
+      <div class="box">
```

Comment:
👀  a box, huh?
This could be a bit more descriptive

Reply:
done